andrew/euler/euler_10.py

```diff
@@ -0,0 +1,28 @@
+'''
+The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.
+Find the sum of all the primes below two million.
+'''
+
+def prime_check2(value):
+    '''Check for prime numbers'''
+    #Declare a set - an unordered collection of unique elements
```

Comment:
These comments are useless. Remove them.

Comments in source code should say 'why' and not 'what' or 'how'. Typically you can say 'why' with function names. Comments in source codes should be rare.